mcs/class/corlib/corlib.dll.sources

```diff
@@ -114,6 +115,8 @@ System/NumberFormatter.cs
 System/Object.cs
 ../../../external/corefx/src/System.Runtime.Extensions/src/System/OperatingSystem.cs
 System/PlatformID.cs
+../../../external/corefx/src/Common/src/CoreLib/System/Range.cs
+../../../external/corefx/src/Common/src/CoreLib/System/ReadOnlySpan.Fast.cs
 ../../../external/corefx/src/Common/src/CoreLib/System/ResolveEventArgs.cs
```

Comment:
Don't we already have `ReadOnlySpan.Fast.cs`  and `Span.Fast.cs` in this file?

Reply:
We did; thanks for spotting this, I fixed it.